sites/fast-website/src/app/components/section-header/section-header.template.ts

```diff
@@ -0,0 +1,11 @@
+import { html } from "@microsoft/fast-element";
+import { SectionHeader } from "./section-header";
+
+export const SectionHeaderTemplate = html<SectionHeader>`
+    <template>
```

Comment:
Should this control have a default slot?

Reply:
Since this text changes in size base where it is used, making it a slot does make sense.

Reply:
Sorry, for clarity - every slot here is named. Is there a default slot for the control? I might make the heading the default slot for the control as I could foresee adding sections that do not have body copy.

Reply:
That makes sense. I will add a default slot